arches_references/management/commands/controlled_lists.py

```diff
@@ -82,6 +82,9 @@ def migrate_collections_to_controlled_lists(
                 -ho 'http://localhost:8000/plugins/controlled-list-manager/item/'
                 -psl 'fr'
                 -ow
+
+            for collections that contain an apostrophe, wrap the concept in double quotes, e.g. "John''s list"
+
```

Comment:
I only get a PL/pgSQL error if I don't provide the overwrite flag. Perhaps I just need to add better error reporting when trying to insert records with the same id (e.g. running migration x2 for same collection, without overwrite), which I had put off for now because it's useful to see the PL/pgSQL error during development.

Just to be certain I'm understanding correctly, repro instructions:
1. add collection with apostrophe
2. call `migrate_collections_to_controlled_lists` (without overwrite flag (?))